html-lists/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Lists</title>
+  </head>
+  <body>
+    <h1>HTML Lists</h1>
+    <h2>Ordered List Example</h2>
+      <ol>
+        <li>Learn to code.</li>
+        <li>Build projects.</li>
+        <li>Get a job.</li>
+      </ol>
+    <h2>Unordered List Example</h2>
+      <ul>
+        <li>HTML</li>
+        <li>CSS</li>
+        <li>JS</li>
+        <li>SQL</li>
+        <li>PHP</li>
+      </ul>
```

Comment:
This is a sibling to the `h2` above it, so it should be at the same level of indentation.